src/libsync/propagatedownload.cpp

```diff
@@ -820,6 +848,37 @@ void PropagateDownloadFile::downloadFinished()
             return;
         }
         qCInfo(lcPropagateDownload) << "Created conflict file" << fn << "->" << conflictFileName;
+
+        // Create a new conflict record. To get the base etag, we need to read it from the db.
+        ConflictRecord conflictRecord;
+        conflictRecord.path = conflictFileName.toUtf8();
+        conflictRecord.baseModtime = _item->_previousModtime;
+
+        SyncJournalFileRecord baseRecord;
+        if (propagator()->_journal->getFileRecord(_item->_originalFile, &baseRecord)) {
+            conflictRecord.baseEtag = baseRecord._etag;
+            conflictRecord.baseFileId = baseRecord._fileId;
+        } else {
+            // We might very well end up with no fileid/etag for new/new conflicts
```

Comment:
getFileRecord returns false only if there is a database error!
If the entry is not in the database, baseRecord.isValid() == false